src/utils/httpClient.ts

```diff
@@ -0,0 +1,11 @@
+const BASE_URL =
+  'https://mate-academy.github.io/react_dynamic-list-of-todos/api/';
+
+export function GetData(url: string) {
+  return fetch(BASE_URL + url).then(response => {
+    if (!response.ok) {
```

Comment:
The `if (!response.ok)` block is empty. Consider adding error handling logic here, such as throwing an error or returning a rejected promise, to handle cases where the response is not successful.